### PR TITLE
Update UIA to UI Automation in UI and also Name Automation Property.

### PR DESCRIPTION
**Pull request by neocln**
#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [x] Does this address an existing issue? If yes, Issue# - #167 
- [x] Includes UI changes?
  - [x] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [x] Attach any screenshots / GIF's that are applicable.

Note - After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge. 

#### Describe the change
Update UIA to UI Automation in UI and Name automation property. 
image

image

image

### CODE REVIEW

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 1]
```diff
@@ -271,22 +271,22 @@
     <value>Start or stop event recording for the selected element.</value>
   </data>
   <data name="lblHkPauseContent" xml:space="preserve">
-    <value>Pause or resume updates to the UIA tree view when focus changes via mouse or keyboard navigation.</value>
+    <value>Pause or resume updates to the UI Automation tree when focus changes via mouse or keyboard navigation.</value>
   </data>
   <data name="lblToParentContent" xml:space="preserve">
-    <value>Inspect the Parent element in UIAutomation tree.</value>
+    <value>Inspect the Parent element in UI Automation tree.</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>Inspect the Parent element in the UI Automation tree.</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 2]
```diff
@@ -271,22 +271,22 @@
...
   </data>
   <data name="lblToBeforeContent" xml:space="preserve">
-    <value>Inspect the Previous sibling element in UIAutomation tree.</value>
+    <value>Inspect the Previous sibling element in UI Automation tree.</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>Inspect the Previous sibling element in the UI Automation tree.</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 3]
```diff
@@ -271,22 +271,22 @@
...
   </data>
   <data name="lblToNextContent" xml:space="preserve">
-    <value>Inspect the Next sibling element in UIAutomation tree.</value>
+    <value>Inspect the Next sibling element in UI Automation tree.</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>Inspect the Next sibling element in the UI Automation tree.</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 4]
```diff
@@ -271,22 +271,22 @@
...
   </data>
   <data name="lblToFirstChildContent" xml:space="preserve">
-    <value>Inspect the First child element in UIAutomation tree.</value>
+    <value>Inspect the First child element in UI Automation tree.</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>Inspect the First child element in the UI Automation tree.</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 5]
```diff
@@ -271,22 +271,22 @@
...
   </data>
   <data name="lblToLastChildContent" xml:space="preserve">
-    <value>Inspect the Last child element in UIAutomation tree.</value>
+    <value>Inspect the Last child element in UI Automation tree.</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>Inspect the Last child element in the UI Automation tree.</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx [hunk 6]
```diff
@@ -436,7 +436,7 @@
     <value>Inspect details about {0}</value>
   </data>
   <data name="tbViewResultsText" xml:space="preserve">
-    <value>View results in UIA tree</value>
+    <value>View results in UI Automation tree</value>
```
Comment (RobGallo) [inline src/AccessibilityInsights.SharedUx/Properties/Resources.resx]:
```suggestion
    <value>View results in the UI Automation tree</value>
```

File: src/AccessibilityInsights.SharedUx/Properties/Resources.resx (hunk 1 above)
Comment (neocln) [reply src/AccessibilityInsights.SharedUx/Properties/Resources.resx]: > Suggested change 
>     <value>Inspect the Parent element in UI Automation tree.</value>
>     <value>Inspect the Parent element in the UI Automation tree.</value>

just this or all others?

**Comment by RobGallo** [comment]
Only the ones I marked.

**Pull request merged**